docker-compose.latest.yml

```diff
@@ -21,6 +21,7 @@ services:
     environment:
       - *chaosgenius-version
       - IN_DOCKER=True
+      - REACT_APP_IS_DEMO=${REACT_APP_IS_DEMO}
```

Comment:
This needs to be added to the other compose files too (and frontend service too).